### DISCUSSION TITLE
키글챌 12회차 반성문

### DISCUSSION THREAD

**Discussion by pnudpwls1103** [BDD-CLUB/challenge-every-week-posting#162] [6. 반성문]
엉엉 TDD 글을 마무리하지 못 했네요.

막 놀진 않았고 졸과 하면서 머리 쥐어 뜯고 있었습니다 ㅎㅎ..
![화면 캡처 2023-09-04 003957](https://github.com/amaran-th/every-week-posting-challenge/assets/68144657/1bdb2d40-d53b-401e-a847-1a509968c7ad)

변명 하나 더 하자면 요즘 주에 1~2번씩 병원 가면서 과거에 체력을 날려버린 죗값을 치르고 있는데요. 몸이 아파서 스트레스 받는 거보다 나는 아프다라는걸 인지하면서 과거의 생활을 돌아보는게 훨씬 스트레스 받네요.. 이번 주는 건강한 삶을 살면서 퇴근 후 공부하는 것도 시작해보겠습니다ㅎㅎ.. 모두 몸 관리 잘 하시고 화이팅!

**Comment by shkisme** [comment]
ㅠㅠㅠㅠ 아프지마세요....! 항상 건강이 최고인 걸 명심해야겠습니당 🥺

**Reply by pnudpwls1103** [reply]
건강관리 화이띵..!!
